docs/DeveloperGuide.md

```diff
@@ -232,22 +193,28 @@ the initial bill to be 0 and chose to remove `Bill` as it would make the `add` c
 * As adding the guest will be done during check in, the guest might not have any special requests to make for the room. Hence,
 we chose to make `Request` optional and default it as blank should it not be provided.
 
-### Edit Command
+### Bill Field
 
 #### Implementation:
-* The `edit` command takes in an INDEX indicating the index of the guest to edit in the current panel (starting from 1) and 8 optional fields (`Name`, `Phone`, `Email`, `Room`, `Date Range`, `Number of Guests`, `Is Room Clean` and `Request`) and is supported by the `EditCommandParser` that extracts out each of the fields from their respective prefixes.
-
-The following activity diagrams summarizes what happens when a user enters an `edit` command.
 
-![EditActivityDiagram](images/EditActivityDiagram.png)
+* The `Bill` class holds the value that a `Guest` is required to pay to the hotel.
+* Its constructor takes in a string representing a signed `double` with up to 2 decimal places.
 
 #### Design Considerations:
 
-**Aspect: Allowing only specific fields provided to be edited**
-* As the edit command is usually used when there is a change or error in the information provided, it makes more sense for the user to be able to change only selected fields.
+**Aspect: `add()` method of the `Bill` class**
+* As `Bill`s can be added to each other, we abstracted this behaviour into the `add` method.
 
-**Aspect: Excluding `Bill` in the `edit` command**
-* As the `bill` command allows us to add and subtract to the bill directly, the edit command is redundant and may cause user error if they were to replace the `Bill` by accident.
+**Aspect: How to deduct from bills**
+* **Alternative 1:** Create a `subtract` method.
+    * Pros: More understandable code.
+    * Cons: Requires knowledge of which method to call (`add` or `subtract`).
+* **Alternative 2 (current choice):** Use `add()` with a negative `Bill`.
+    * Pros: Less code, more flexibility.
+    * Cons: `Bill` must be allowed to hold negative values, but the bill field for each `Guest`
+      cannot, thus requires more checks.
```

Comment:
Should this be "requiring" instead of "requires"?

Reply:
fixed!